src/main/java/seedu/duke/model/ItemContainer.java

```diff
@@ -188,4 +190,25 @@ public String toString() {
     public int getSize() {
         return items.size();
     }
+
+    /**
+     * Gets description of item.
+     *
+     * @param selectedItem Item which description is retrieved from
+     * @return String of description
+     */
+    public String getDescription(Item selectedItem) {
+        String temp = "";
+        String name = selectedItem.getName();
+        BigDecimal sellingPrice = selectedItem.getSellingPrice();
+        BigDecimal purchaseCost = selectedItem.getPurchaseCost();
+
+        temp += "name: " + name + "\nselling price: " + sellingPrice
```

Comment:
It will be better if you can create a constant string, and use String.format() to format the output.
You may refer to https://github.com/AY2122S1-CS2113T-F11-4/tp/blob/master/src/main/java/seedu/duke/Parser.java#L53